### DISCUSSION TITLE
All parameters shown on requests on register

### DISCUSSION THREAD

**Discussion by CamilleBarnier** [fastapi-users/fastapi-users#1454] [Q&A]
## Describe the bug

All parameter of user create is shown on register route
![image](https://github.com/user-attachments/assets/6cdaad46-9c46-47dc-9fd8-d827d02fb0d6)

## Expected behavior

Only mail, password, username and role should be shown

## Configuration
 - Python version :3.10.12
 - fastapi==0.112.2
 - fastapi-users==13.0.0

### FastAPI Users configuration

```py
app.include_router(
    fastapi_users.get_register_router(user_schema=UserRead, user_create_schema= UserCreate),
    prefix="/auth",
    tags=["auth"],
)

class UserCreate(schemas.BaseUserCreate):
    """
    UserCreate class for creating a new user.
    Attributes:
        username (str): The username of the user.
        role (UserRole): The role assigned to the user. Defaults to UserRole.FREE.
    """

    username: str
    role: UserRole = UserRole.FREE

```

**Comment by hgalytoby** [comment]
fastapi-users is designed in such a way that you don't need to worry about other parameters of the register route, which will be excluded during execution.

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/register.py#L57

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L135

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/schemas.py#L34

**Comment by CamilleBarnier** [comment]
Thanks for the support.
